sr_demos/scripts/sr_demos/demo.py

```diff
@@ -114,15 +127,15 @@ def confirm_touched(self):
 
 
 class KeyboardPressDetector:
-    def __init__(self, hand_commander, demo_states, tactile_reading, hand_type):
+    def __init__(self, robot_type):
```

Comment:
Add pydoc please!

Reply:
fixed in 92637be0ff846ca31b76aae33b809ec2e931900b